notebooks/234-blip/Untitled.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Should we say that for QA you need one function and for captioning another? This looks a bit long, but only some parts are needed for specific task, right?</p><p>Also, I think we have opportunity to talk a bit about sequence of functions, i.e. image_encoder+decoder for captioning, image_encoder+text_encoder+decoder for QA</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I'll add</p>

Reply:
added